WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -93,6 +93,14 @@ def index
       end
     end
 
+    begin
+      from_date = params[:from_date] && Date.parse(params[:from_date])
+      @competitions = @competitions.where("start_date >= ?", from_date) if from_date
```

Comment:
That's nice, but since Rubocop complains about the empty `rescue` I'm going to use @jfly's suggestion =)